test/itemlist_formaction.cpp

```diff
@@ -460,8 +460,80 @@ TEST_CASE("OP_BOOKMARK works properly", "[itemlist_formaction]") {
 
 }
 
-TEST_CASE("process_op(OP_EDITFLAGS)", "[itemlist_formaction]") {
-	//NOTIMPL
+TEST_CASE("OP_EDITFLAGS arguments are added to an item's flags", "[itemlist_formaction]") {
+	controller c;
+	newsbeuter::view v(&c);
+	configcontainer * cfg = c.get_cfg();
+	cache rsscache(":memory:", cfg);
+
+	std::vector<std::string> op_args;
+
+	v.set_config_container(cfg);
+	c.set_view(&v);
+
+	std::shared_ptr<rss_feed> feed = std::make_shared<rss_feed>(&rsscache);
+	std::shared_ptr<rss_item> item = std::make_shared<rss_item>(&rsscache);
+
+	itemlist_formaction itemlist(&v, itemlist_str);
+
+	feed->add_item(item);
+	itemlist.set_feed(feed);
+	
+	SECTION("Single flag"){
+		std::string flags = "G";
+		op_args.push_back(flags);
+	
+		REQUIRE_NOTHROW(itemlist.process_op(OP_EDITFLAGS, true, &op_args));
+		REQUIRE(item->flags() == flags);
+	}
+
+	SECTION("Unordered flags"){
+		std::string flags = "abdefc";
+		std::string ordered_flags = "abcdef";
+		op_args.push_back(flags);
+	
+		REQUIRE_NOTHROW(itemlist.process_op(OP_EDITFLAGS, true, &op_args));
+		REQUIRE(item->flags() == ordered_flags);
+	}
+
+	SECTION("Duplicate flag in argument"){
+		std::string flags = "Abd";
+		op_args.push_back(flags + "ddd");
+	
+		REQUIRE_NOTHROW(itemlist.process_op(OP_EDITFLAGS, true, &op_args));
+		REQUIRE(item->flags() == flags);
+	}
+
+	SECTION("Unauthorized values in arguments"){
+		std::string flags = "Abd";
+		SECTION("Numbers"){
+			op_args.push_back(flags + "1236");
+	
+			REQUIRE_NOTHROW(itemlist.process_op(OP_EDITFLAGS, true, &op_args));
+			REQUIRE(item->flags() == flags);
+		}
+		SECTION("Symbols"){
+			op_args.push_back(flags + "%^\\*;\'\"&~#{([-|`_/@)]=}$£€µ,;:!?./§");
```

Comment:
"Symbol" is a vague word; I think you should use `LOG` here to display what symbols you add—it will help understand what's wrong if the test fails.